bin/pip3

```diff
@@ -1,4 +1,6 @@
-#!/Users/jaebradley/projects/basketball_reference_web_scraper/bin/python3
```

Comment:
@PGatts these `bin` files should never have been committed - I removed them in https://github.com/jaebradley/basketball_reference_web_scraper/commit/81dd1e0fd50aac300361a18d7b893b12183f6f4b

Rebasing / merging the latest changes in `v4` should resolve the merge conflicts caused by the `bin` directory.